examples/standalone/gtest_setup/CMakeLists.txt

```diff
@@ -31,6 +31,11 @@ foreach(TEST_TARGET
     gtest_main
     gz-sim::gz-sim
   )
+
   include(GoogleTest)
-  gtest_discover_tests(${TEST_TARGET})
+  if (APPLE)
```

Comment:
Do you think this issue will occur on other platforms with cmake 4? If so, I would check the cmake version instead of APPLE

I'm not sure though

Reply:
I did some tests:
* Ubuntu + cmake 4.0.1: built fine
* Mac + cmake 3.31.7: same error
* Mac + cmake 3.31.6: same error

hmm maybe something else has changed

Reply:
weird

I've tried disabling ccache and that seems to fix it for me?

testing without ccache using https://github.com/gazebo-tooling/release-tools/commit/b7cd4e3681d81628613e6bdd3051c58f13cfee0a:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_sim-ci-gz-sim9-homebrew-amd64&build=97)](https://build.osrfoundation.org/job/gz_sim-ci-gz-sim9-homebrew-amd64/97/) https://build.osrfoundation.org/job/gz_sim-ci-gz-sim9-homebrew-amd64/97/

Reply:
that didn't help

Reply:
now I can't reproduce the failure on my personal machine at all, though it still fails in CI

Reply:
hmm I can still reproduce it. I don't have ccache setup.